js/views/storeWizardVw.js

```diff
@@ -148,12 +150,19 @@ module.exports = baseModal.extend({
   },
 
   renderModerator: function(moderator){
+    //make sure this moderator is not a duplicate
```

Comment:
Have we logged a bug on the server so this is fixed at the source? This is really another case where the client shouldn't be accounting for what's really a defect in the server.